lib/menu.rb

```diff
@@ -0,0 +1 @@
+{fish_and_chips: 5.99, curry_and_rice: 7.99, wun_ton_soup: 5.99, sweet_and_sour_pork: 6.50, prawns_chop_suey: 5.50}
```

Comment:
no class here? If you are intending to hold an external state, you could use YAML, JSON, CSV or even plain text. I would expect a .rb file to be executable.